src/components/biddingPage/submit/addBid.tsx

```diff
@@ -0,0 +1,258 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import dayjs from 'dayjs';
+
+interface IProps {
+  type: string;
+}
+
+const AddBid: React.FC<IProps> = ({ type }) => {
+  // date
+  const [date, setDate] = useState<string>('null');
+
+  // time
+  const [time, setTime] = useState<number>(-1);
+
+  // volume
+  const [volume, setVolume] = useState<number>(0);
+
+  // price
+  const [price, setPrice] = useState<number>(0);
+
+  // total price
+  const [totalPrice, setTotalPrice] = useState<number>(0);
+
+  // set submit button disabled
+  const [submitDisabled, setSubmitDisabled] = useState<boolean>(true);
+
+  // click reset btn or not
+  const [reset, setReset] = useState<boolean>(true);
+
+  // images
+  const [checkImg, setCheckImg] = useState<string>('check-gray');
+  const [resetImg, setResetImg] = useState<string>('cancel-gray');
+
+  // create an array from '0:00 - 1:00' to '23:00 - 24:00'
+  const intervalArr: string[] = [
```

Comment:
Since you have define this array in many files, I think it's better to put this in an file and import it.
Can you create `src/constants/constant.ts` to store all this reusable variables?

Reply:
Done. Thanks !